handlers.go

```diff
@@ -218,13 +221,33 @@ func setupGatewayHandler(cfg Config, nd *Node, tracingAuth string) (http.Handler
 			if request.Header.Get("Tracestate") != "" {
 				request.Header.Del("Tracestate")
 			}
+			if request.Header.Get(NoBlockcacheHeader) != "" {
+				request.Header.Del(NoBlockcacheHeader)
+			}
+		}
+
+		// Process cache skipping header
+		if noBlockCache := request.Header.Get(NoBlockcacheHeader); noBlockCache != "" {
+			ds, err := leveldb.NewDatastore("", nil)
+			if err != nil {
+				writer.WriteHeader(http.StatusInternalServerError)
+				_, _ = writer.Write([]byte(err.Error()))
+				return
+			}
+			newCtx := context.WithValue(request.Context(), NoBlockcache{}, blockstore.NewBlockstore(ds))
+			request = request.WithContext(newCtx)
 		}
+
 		prevHandler.ServeHTTP(writer, request)
 	})
 
 	return handler, nil
 }
 
+const NoBlockcacheHeader = "Rainbow-No-Blockcache"
```

Comment:
Up for better suggestions on the header name

Reply:
Maybe a generic `Rainbow-Skip-Caches`? 
We may want to skip things like namesys cache as well at some point too.

Reply:
After sleeping on this, having an explicit flag for each system is more useful for debugging, let's keep as-is.